include_core/omrutilbase.h

```diff
@@ -37,6 +37,7 @@ void issueReadWriteBarrier(void);
 void issueWriteBarrier(void);
 uintptr_t addAtomic(volatile uintptr_t *address, uintptr_t addend);
 uintptr_t subtractAtomic(volatile uintptr_t *address, uintptr_t value);
+uintptr_t setAtomic(volatile uintptr_t *address, uintptr_t value);
```

Comment:
Could we add Doxygen documentation to the newly added API. There is an example of this a few lines below.

Reply:
Added Doxygen documentation.